gulpfile.js

```diff
@@ -83,44 +71,70 @@ gulp.task('fonts', function () {
     .pipe($.size({title: 'fonts'}));
 });
 
+
 // Compile and automatically prefix stylesheets
 gulp.task('styles', function () {
+
+  var AUTOPREFIXER_BROWSERS = [
+    'ie >= 10',
+    'ie_mob >= 10',
+    'ff >= 30',
+    'chrome >= 34',
+    'safari >= 7',
+    'opera >= 23',
+    'ios >= 7',
+    'android >= 4.4',
+    'bb >= 10'
+  ];
+
+
+
   // For best performance, don't add Sass partials to `gulp.src`
   return gulp.src([
-    'app/styles/*.scss',
-    'app/styles/**/*.css',
-    'app/styles/components/components.scss'
+    'app/**/*.scss',
+    'app/styles/**/*.css'
   ])
-    .pipe($.sourcemaps.init())
     .pipe($.changed('.tmp/styles', {extension: '.css'}))
+    .pipe($.sourcemaps.init())
     .pipe($.sass({
       precision: 10,
       onError: console.error.bind(console, 'Sass error:')
     }))
-    .pipe($.autoprefixer({browsers: AUTOPREFIXER_BROWSERS}))
-    .pipe($.sourcemaps.write())
-    .pipe(gulp.dest('.tmp/styles'))
+    .pipe($.autoprefixer(AUTOPREFIXER_BROWSERS))
+    .pipe(gulp.dest('.tmp'))
     // Concatenate and minify styles
     .pipe($.if('*.css', $.csso()))
-    .pipe(gulp.dest('dist/styles'))
+    .pipe($.sourcemaps.write())
+    .pipe(gulp.dest('dist'))
     .pipe($.size({title: 'styles'}));
+})
+
+// Concatenate and minify JavaScript
+gulp.task('scripts', function () {
```

Comment:
I'm unsure which style guide we use but this does seem inconsistent: https://github.com/google/web-starter-kit/pull/676/files#diff-b9e12334e9eafd8341a6107dd98510c9L168.